Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/salvageweapons.yml

```diff
@@ -0,0 +1,57 @@
+- type: entity
+  parent: MarkerBase
+  id: SalvageWeaponSpawner
+  name: Salvage Weapon Spawner
+  components:
+  - type: Sprite
+    layers:
+      - state: red
+      - sprite: Objects/Weapons/Guns/Snipers/bolt_gun_wood.rsi
+        state: base
+  - type: RandomSpawner
+    offset: 0
+    prototypes:
+      - WeaponSniperMosin
```

Comment:
I removed most of the Syndicate uplink items in the last PR regarding Salvage. Please do not re-add them.

Reply:
Mosin should be added back in and further discussion should be had in blurring the lines of "Uplink" guns to where they don't immediately out you--Such as how MK was made an uplink gun over the Cobra.

Reasoning: Helps cut down on metagaming and introduces roleplay, while also curbing some of the Red vs Blue mentality folks have.

This doesn't mean there can't be "traitor weapons" that out you, such as the e-sword or even the predator, but its better to blur the lines--especially for poor weapons like the mosin.

Reply:
>  further discussion should be had in blurring the lines of "Uplink" guns to where they don't immediately out you

yes